extra/org.sil.Bloom.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.0
+Name=Bloom Alpha
+Comment=Literacy materials development for language communities
+Exec=org.sil.Bloom %u
```

Comment:
manifest say ` command: bloom`

So this is incorrect.

Reply:
I changed to using the .desktop and .appdata.xml from the BloomDesktop/debian folder.  I updated the .appdata.xml and updated the commit value to get the updated file.